maestro/lifecycle.py

```diff
@@ -17,7 +17,7 @@
 class BaseLifecycleHelper:
     """Base class for lifecycle helpers."""
 
-    def test(self):
+    def test(self, ship=None, container=None):
```

Comment:
There is. Line 610 of `maestro/entities.py`, when you do the `pool.map_async`:

```python
return pool.map_async(lambda check: check.test(self), self._lifecycle[state])
```

This will pass the `Container` instance to the check's test() method.